src/Resources/contao/templates/forms/form_captcha.html5

```diff
@@ -15,4 +15,16 @@
 
   <input type="text" name="<?= $this->name ?>" id="ctrl_<?= $this->id ?>" class="captcha mandatory<?php if ($this->class) echo ' ' . $this->class; ?>" value=""<?= $this->getAttributes() ?>>
   <span class="captcha_text<?php if ($this->class) echo ' ' . $this->class; ?>"><?= $this->getQuestion() ?></span>
+
+  <?php if (!$this->hasErrors()): ?>
+    <div style="display:none">
+      <label for="ctrl_<?= $this->id ?>_hp">Do not fill out this field</label>
```

Comment:
With `display:none`, the message will never be read out by screen readers. We do not need to translate it therefore.